_js/highlight/.travis.yml

```diff
@@ -0,0 +1,30 @@
+language: node_js
+node_js:
+  - "4"
+  - "5"
+env:
+  -
+  - BROWSER=1
+  - BROWSER=1 NOCOMPRESS=1
+script:
```

Comment:
Maybe a better place for this might be `scripts/highlight`, WDYT?

Reply:
you can hoist all the dependencies to devdep in the main one I think, or use yarn workspaces

Reply:
I'll try to use the built version without submodules. Seems that netlify build process can't run git commands. Maybe I'm wrong

Reply:
Can i see travis logs?

Reply:
the logs are available if you click on the error normally

Reply:
https://app.netlify.com/sites/yarnpkg/deploys/5a09b523a114770371f9c8fc for example